docs/_includes/common/page-notice.html

```diff
@@ -57,75 +57,75 @@ <h3 class="page-notice__title">Title copy goes here</h3>
                     </svg>
                 </div>
                 <div class="page-notice__main">
-                    <h3 class="page-notice__title">Title copy goes here</h3>
+                    <h3>Congratulations! Your item has sold!</h3>
                     <p>Details...</p>
                 </div>
                 <div class="page-notice__footer">
-                    <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
+                    <button class="fake-link fake-link--action">Action</a>
                 </div>
             </section>
         </div>
     </div>
     {% highlight html %}
-<section class="page-notice page-notice--confirmation" role="region" aria-label="Confirmation">
-    <div class="page-notice__header">
-        <svg class="icon icon--confirmation-filled" focusable="false" height="24" width="24" role="img" aria-label="Confirmation">
-            <use xlink:href="#icon-confirmation-filled"></use>
-        </svg>
-    </div>
-    <div class="page-notice__main">
-        <h3 class="page-notice__title">Title copy goes here</h3>
-        <p>Details...</p>
-    </div>
-    <div class="page-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
-    </div>
-</section>
-
-<section class="page-notice page-notice--attention" role="region" aria-label="Attention">
-    <div class="page-notice__header">
-        <svg class="icon icon--attention-filled" focusable="false" height="24" width="24" role="img" aria-label="Attention">
-            <use xlink:href="#icon-attention-filled"></use>
-        </svg>
-    </div>
-    <div class="page-notice__main">
-        <h3 class="page-notice__title">Title copy goes here</h3>
-        <p>Details...</p>
-    </div>
-    <div class="page-notice__footer">
-        <a href="https://www.ebay.com" class="fake-btn fake-btn--secondary fake-btn--transparent">Action</a>
-    </div>
-</section>
+    <section class="page-notice page-notice--confirmation" role="region" aria-label="Confirmation">
```

Comment:
Looks like the formatting went a little awry:

![Screen Shot 2021-06-16 at 4 38 31 PM](https://user-images.githubusercontent.com/38065/122308577-7ca30300-cec1-11eb-9c3f-91cb88ece306.png)